autoPyTorch/pipeline/components/training/metrics/base_metric.py

```diff
@@ -0,0 +1,25 @@
+from typing import Any, Dict, Optional
+
+from pytorch_lightning.metrics.metric import Metric
+
+from sklearn.base import BaseEstimator
+
+import torch.tensor
+
+
+class autoPyTorchMetric(BaseEstimator):
+    def __init__(self):
+        self.metric: Optional[Metric] = None
```

Comment:
I think we need a mechanism to ensure that the optimizer (BOHB, SMAC, etc) can tell if this metric is better the smaller it gets or better if it goes higher.

Could you add an attribute like that?